frontend/src/lib/components/LemonTag/LemonTag.scss

```diff
@@ -30,11 +29,17 @@
         color: var(--light);
     }
 
+    &.purple {
+        background-color: #ebe3ed;
+        color: var(--indigo);
+    }
```

Comment:
Hmm, `purple` doesn't seem to fit in conceptually – it's a tag type, but the existing types are `'highlight' | 'warning' | 'danger' | 'success' | 'default'`. None of them is a color.

Also, this introduces a color that's not defined as a variable in `root-variables` – this is dangerous as we can easily introduce dozens of slightly different colors variations this way, which is hard to keep track of! Pretty much all colors should be defined at the top level.